### ISSUE TITLE
Some fields are missing in the SDK

### ISSUE THREAD

**Issue by rjanits** [justimmo/php-sdk#153]
We implement a website for a customer that pulls the objects from Justimmo and displays them in a custom layout. According to the design, some fields have been defined by Justimmo but are not available via the SDK.

For example many Area/Flächen fields:

- verkaufsflaeche
- verbaubare_flaeche
- hallenhoehe
- raumhoehe
- mietdauer_ ...
- befristung,
- bauart
- ...

When is it planned that the missing fields are also available via the SDK?

**Comment by Cashewz** [comment]
Hi rjantism,

thanks for your feedback. We will add the missing fields over the next few weeks.
Please access the fields directly through the API instead of using the SDK if you need them sooner.

**Comment by 19Gerhard85** [comment]
@rjanits The missing files where added in 1.2.6

**Comment by rjanits** [comment]
> @rjanits The missing files where added in 1.2.6

Awesome. Thank you very much.

**Comment by zoltan-sule** [comment]
**These fields were available before 1.2.6**

- verbaubare_flaeche - Verbaubare Fläche --> buildableArea
- raumhoehe          - Raumhöhe          --> ceilingHeight
- bauart_id          - Bauart            --> styleOfBuildingId


**New fields from 1.2.6**

- verkaufsflaeche                     -> salesArea
- hallenhoehe                           -> hallHeight
- udsf_befristete_flaeche         -> shortTermArea
- udsf_gewichtete_flaeche       -> weightedArea
- udsf_rohdachboden_flaeche -> undevelopedAtticArea

Fixed RentDurationType:
- it supports unlimited (unbefristet) renting period

```
<?php

require __DIR__ . "/../vendor/autoload.php";

use Justimmo\Api\JustimmoApi;
use Justimmo\Model\RealtyQuery;
use Justimmo\Model\Wrapper\V1\RealtyWrapper;
use Justimmo\Model\Mapper\V1\RealtyMapper;

$api = new JustimmoApi('*********', '*********');

$mapper  = new RealtyMapper();
$wrapper = new RealtyWrapper($mapper);

$query = new RealtyQuery($api, $wrapper, $mapper);

try {
    $realties = $query
        ->filterByStatusId(5)
        ->filterByBuy(0)
        ->filterByRent(1)
        ->setLimit(1)
        ->find();

    echo sprintf("%d objects<br/><br/>", $realties->count());

    /** @var \Justimmo\Model\Realty $realty */
    foreach ($realties as $realty) {
        echo sprintf('oid/number: %d/%d (%s): %s<br/>', $realty->getId(), $realty->getPropertyNumber(), $realty->getUpdatedAt(), $realty->getTitle());
        echo sprintf('TotalArea / SalesArea / temporaryArea / weightedArea / rawAtticArea: %f, %f, %f, %f, %f<br/>', $realty->getTotalArea(), $realty->getSalesArea(), $realty->getShortTermArea(), $realty->getWeightedArea(), $realty->getUndevelopedAtticArea());
        echo sprintf('BuildableArea / styleOfBuilding: %f, %s<br/>', $realty->getBuildableArea(), $realty->getStyleOfBuilding());
        echo sprintf('ceilingHeight / hallHeight: %f, %f, <br/>', $realty->getCeilingHeight(), $realty->getHallHeight());
        echo sprintf('RentDuration / RentDurationType, %f, %s<br/><br/>', $realty->getRentDuration(), $realty->getRentDurationType());
        //var_export($realty);
        break;
    }

} catch (Throwable $e) {
    echo $e->getMessage();
}
```
